generated/ruby/google-cloud-ruby/google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1.rb

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
I agree, but the reason why I kept it here because in order to run the modified integration test (with new way to call 'require'), I have to have the v1beta1.rb here.